src/ucp/core/ucp_ep.c

```diff
@@ -67,6 +67,7 @@ void ucp_ep_add_to_hash(ucp_ep_h ep)
         ucs_fatal("Hash failed with ep %p to %s 0x%"PRIx64"->0x%"PRIx64
                   "with status %d", ep, ucp_ep_peer_name(ep), worker->uuid,
                   ep->dest_uuid, hash_extra_status);
+        return;
```

Comment:
i need to think more about this approach since it means that now all code places which do ucs_fatal() cannot assume it it will not return